### PR TITLE
Increasing coverage for tests with Postgres as sink database

### PR DESCRIPTION
**Pull request by vaibhav-yb**
This PR aims to add more scenarios where different kinds of operations are performed on the source YugabyteDB database and then it is verified that the operations are reflected across the sink database. The following scenarios are added:
* `INSERT-->UPDATE-->DELETE` verifying the sink database also ends up with zero records.
* `INSERT-->UPDATE` to verify that the updates are being propagated properly.
* Batch inserts of records to verify that the intents are being read and propagated properly.
* Inserting 30k records in a single transaction using `generate_series()`
* Inserting few records, then starting the CDCSDK server to verify that the tablet is being bootstrapped and only the records after bootstrapping are being streamed.

### CODE REVIEW

**Comment by suranjan** [comment]
Please elaborate what are the 'more scenarios' !

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java [hunk 1]
```diff
@@ -86,13 +87,13 @@ public static void beforeAll() throws Exception {
         containerNetwork = Network.newNetwork();
 
         kafkaContainer = new KafkaContainer(DockerImageName.parse(KAFKA_IMAGE))
-                            .withNetworkAliases("kafka")
-                            .withNetwork(containerNetwork);
+                .withNetworkAliases("kafka")
```
Comment (suranjan) [inline cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java:90]: Lost of formatting changes? Did we use new formatter?

Comment (suranjan) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java:90]: If there is one project level or mvn based format plugin, we should use that.

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
+import io.debezium.testing.testcontainers.DebeziumContainer;
+
+/**
+ * Release test that verifies reading of multiple combination of operations from 
+ * a YugabyteDB database and writing to Kafka and then further to a PostgreSQL sink database 
+ *
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+public class MultiOpsPostgresSinkConsumerIT {
```
Comment (suranjan) [inline cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java:36]: More test cases if not covered already
1. Single shard/multi shard tx
2. Multi tables
3. Multiple tablets
4. Multiple tasks
5. Restart/bootstrap
6. Composite key
7. data types (ALL have we covered? e.g multi dimensional arrays and all)
8. Unsupported data types

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java (hunk 1 above)
Comment (vaibhav-yb) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/ElasticsearchSinkConsumerIT.java:90]: There is one mvn based format plugin, this formatting was a result of using that only. I will revert these changes and will take them up in the other refactoring PR I will raise.

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java (hunk 2 above)
Comment (vaibhav-yb) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java:35]: I have modified the existing test for multiple tablets and added a new one to test bootstrap i.e. `YBBootstrapPostgresSinkConsumerIT`. For multi table setup or tests involving composite keys, some infra changes will be required - I have started working on the refactoring stuff and these remaining tests will be added a part of the next PR only.

The datatype tests are covered in the tests for the `debezium-connector-yugabytedb` repo.

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                // Clear batch for next iteration
+                st.clearBatch();
+            }
+        }
+        catch (SQLException e) {
+            throw e;
+        }
+
+        // Wait for records to be replicated across Postgres
+        Thread.sleep(5000);
```
Comment (vrajat) [inline cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java:190]: Another simple option is to use `TestHelper.assertRecordCountInPostgres(totalRowsToBeInserted, pgContainerIp);` within Awaitility fn. here.

**Review by vrajat — CHANGES_REQUESTED** [review]

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java (hunk 3 above)
Comment (vaibhav-yb) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java:190]: I am doing that as a part of my next refactoring PR.

**Review by vrajat — APPROVED** [review]

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java (hunk 2 above)
Comment (suranjan) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java:35]: I hope above test cases are tracked for later.

**Pull request merged**